src/ipa_pc/data_structures.rs

```diff
@@ -1,48 +1,104 @@
+//! Data structures for the dlog commitment scheme from [BCMS20]:
+//! * [`Parameters`], [`VerifierKey`], [`CommitterKey`]
+//! * [`Commitment`], [`Randomness`], [`Proof`], [`MultiPointProof`]
+//! and the implementation of their corresponding traits from [data_structures.rs](/src/data_structures.rs).
+//! In addition scheme-specific structs [`SuccinctCheckPolynomial`] and [`VerifierState`] are
+//! introduced.
+//!
+//! [BCMS20]: https://eprint.iacr.org/2020/499
 use crate::*;
 use crate::{PCCommitterKey, PCVerifierKey, Vec};
-use algebra::{AffineCurve, Field, PrimeField, ProjectiveCurve, ToBytes, UniformRand};
-use rand::thread_rng;
-use rand_core::RngCore;
+use algebra::{Curve, PrimeField};
 use std::{
     convert::TryFrom,
     io::{Read, Write},
     vec,
 };
 
-/// `UniversalParams` are the universal parameters for the inner product arg scheme.
+// #[cfg(test)]
+use rand::thread_rng;
+// #[cfg(test)]
+// use algebra::UniformRand;
```

Comment:
Why the compiler directives are commented ? Also, if these imports are used just for test, put them directly inside the corresponding mod test {}

Reply:
This is deprecated and removed in monorepo

Reply:
Either you move these changes here and report them on ginger-lib monorepo branch,
or we formally decide to address this changes in ginger-lib, but you need to create a separate branch from the monorepo branch containing them.
Currently, it's difficult to review them as in the monorepo branch everything will appear as new.

Reply:
Done